src/lib/journey/callbacks/password/validated-create-password.svelte

```diff
@@ -5,11 +5,26 @@
   import { getPasswordValidationFailureText, isInputRequired } from '$journey/utilities/callback.utilities';
 
   export let callback: ValidatedCreatePasswordCallback;
+  export let firstInvalidInput: boolean;
   export let idx: number;
 
+  /**
+   * This callback does not indicate that the input is required.
+   * TODO: Can we just assume a password is always required?
+   */
```

Comment:
Sorry, I need to remove this comment. The answer is no, we can't assume password is always required. The below `isInputRequired` will always return false for this callback. The only policy on the callback is the following:

```
policyRequirements: ["VALID_TYPE"]
```

`"VALID_TYPE"` is not very informative, and there doesn't seem to be a way to mark it as required in the Admin UI 🤷  So, I guess this will always slip through and the server will have to validate.

Reply:
Do we have plans to utilise the "real time" validation of the password the user is creating, as the Platform Login UI does? This seems to submit the form using the **validateOnly** flag. I think this is a nice feature to include but I don't consider it blocking to creation of the alpha version. @cerebrl thoughts?